app-emulation/crun/crun-0.16.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit autotools python-any-r1
+
+DESCRIPTION="A fast and low-memory footprint OCI Container Runtime fully written in C"
+HOMEPAGE="https://github.com/containers/crun"
+SRC_URI="https://github.com/containers/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+ LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64"
+IUSE="+bpf +caps criu +seccomp systemd static-libs"
```

Comment:
Btw we are trying to get rid of `static-libs` where they're not needed.

Reply:
what's the reason? I always thought it a strong point of gentoo to be able to get static libraries easily when I need them...

Reply:
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0302